<launch>
    <node pkg="ptu_laser_assembler" type="ptu_scan_assembler" name="ptu_scan_assembler" output="screen">
        <param name="odom_frame" type="string" value="/odom"/>
        <param name="sensor_frame" type="string" value="/ptu_pan_fixation"/>
        <param name="oscPan" type="bool" value="true"/>
        <param name="oscTilt" type="bool" value="false"/>
        <param name="minPan" type="double" value="-1.57"/>
        <param name="maxPan" type="double" value="1.57"/>
        <param name="minTilt" type="double" value="-0.8"/>
        <param name="maxTilt" type="double" value="0.5"/>
        <param name="velocityPan" type="double" value="1.0"/>
        <param name="velocityTilt" type="double" value="0.5"/>
        <param name="msgDelay" type="double" value="0.053"/> <!-- This value was experimentaly computed for husky cpr-lav01 -->
    </node>
</launch>
